### PR TITLE
Release 2.4 RC1 - tagged meta-parsec, meta-mbed-edge and meta-pelion-edge with latest dev SHAs

### PR DESCRIPTION
**Pull request by ygoyal18**

### CODE REVIEW

**Review by petedyerarm — APPROVED** [review]
LGTM

File: pelion.xml [hunk 1]
```diff
@@ -25,5 +25,5 @@
   <project name="meta-parsec"
            path="layers/meta-parsec"
            remote="PelionIoT"
-           revision="337ca136c6ac785433ccec56fa1cacbe956049b0" />
+           revision="fb5b6b1221c0ba9717d384513341c7af41aacd26" />
```
Comment (ygoyal18) [inline pelion.xml:28]:
```suggestion
           revision="0b7cf966e6304b72425f74ea3e89f669c8809006" />
```

**Comment by esajaa** [comment]
Is `dev` correct target for this PR?

**Review by OPpuolitaival — APPROVED** [review]

**Comment by ygoyal18** [comment]
The PR checks are passing after reverting meta-parsec to 0.6.0 parsec service. We now have a fix for 0.7.2, updating meta-parsec SHA to test that out.

File: pelion.xml [hunk 2]
```diff
@@ -25,5 +25,5 @@
   <project name="meta-parsec"
            path="layers/meta-parsec"
            remote="PelionIoT"
-           revision="337ca136c6ac785433ccec56fa1cacbe956049b0" />
+           revision="0b7cf966e6304b72425f74ea3e89f669c8809006" />
```
Comment (ygoyal18) [inline pelion.xml:28]:
```suggestion
           revision="7c8b8d5ee3032c85a96054192f3184f4a9b2a369" />
```

**Pull request merged**